buffer/src/main/java/io/netty5/buffer/BufferInputStream.java

```diff
@@ -252,11 +253,9 @@ public int skipBytes(int n) throws IOException {
     }
 
     /**
-     * Expose the internally owned buffer.
-     * Use only for testing purpose.
-     *
```

Comment:
I think we can keep the comment, since the annotation might not be as visible to people.